assets/texts/en/amoDescription.html

```diff
@@ -1 +1,103 @@
+	<li><code>&gt;=</code> is corrected into <code>≥</code></li>
+	<li><code>1/4</code> is corrected into <code>¼</code></li>
+	<li><code>--></code> is corrected into <code>⟶</code></li>
+	<li><code>--></code> is corrected into <code>⟶</code></li>
+	<li><code><==></code> is corrected into <code>⟺</code></li>
+	<li>and much more…</li>
+</ul>
+
+<b>📢 More Features 📢</b>
+
+<ul>
+	<li>Unicode autocorrection as you type:<ul>
+			<li>Autocorrect <a href="https://en.wikipedia.org/wiki/Miscellaneous_Symbols">Unicode symbols</a> (i.e. hyphens <code>--</code> with –, fractions <code>1/4</code> with ¼, etc.). Supports more
+				than 85 symbols.</li>
+			<li>Use Unicode smart quotes ('single quotes' with ‘Unicode single quotes’ and "double quotes" with “Unicode double quotes”)</li>
+			<li>Convert <a href="https://en.wikipedia.org/wiki/Number_Forms">fractions</a> and <a href="https://en.wikipedia.org/wiki/Mathematical_constant">mathematical constants</a> to Unicode characters (i.e. <code>1234.25</code>
+				with 1234¼, etc.). Supports all Unicode fraction characters, Pi and e constants.</li>
+			<li>Supports undoing autocorrections</li>
+		</ul>
+	</li>
+	<li>Context menu with selected/highlighted text:<ul>
+			<li><a href="https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols">Unicode font</a> conversion (i.e. ꜱᴍᴀʟʟ ᴄᴀᴘꜱ, 𝒮𝒸𝓇𝒾𝓅𝓉, 𝔉𝔯𝔞𝔨𝔱𝔲𝔯, etc.). Supports more than 20 fonts.</li>
+			<li>Change <a href="https://en.wikipedia.org/wiki/Letter_case#Case_styles">casing</a> (i.e. UPPER CASE, lower case, tOGGLE cASE, etc.). Supports all locales.</li>
+			<li>Optionally show a live preview.</li>
+		</ul>
+	</li>
+	<li>Support websites as best as possible by providing website-specific possibles, if needed. (<a href="https://github.com/rugk/unicodify/issues">if there is a problem, please report it</a>)</li>
+	<li>Very fast, even when typing on large documents.</li>
+	<li>Supports the light/dark mode of your system automatically.</li>
+	<li>Settings automatically synced between all browser instances and devices (Firefox and Chrome only).</li>
+	<li>Follows the <a href="https://design.firefox.com/photon">Firefox</a> and <a href="https://style.thunderbird.net/">Thunderbird</a> Photon Design.</li>
+	<li>Puts your privacy first and does not send data anywhere.</li>
+	<li>Compatible with Firefox for Android.</li>
+	<li>Translated into several languages. <a href="https://github.com/rugk/unicodify/blob/main/CONTRIBUTING.md#Translations">Contribute your own language!</a></li>
+	<li>Settings can be managed by your administrator.</li>
+</ul>
+
+
+
+<b>📝 Development 📝</b>
+The add-on is free/libre open-source software and developed on GitHub. <a href="https://github.com/rugk/unicodify">Fork it on GitHub</a> and contribute.
+<a href="https://github.com/rugk/unicodify/contribute">There are some easy issues to start with.</a>
+
+This extension only works with modern Firefox and Thunderbird v78 or higher.
+
+
+<b>🐬 Contribute 🐬</b>
```

Comment:
Maybe use one of these emojis instead: 🙋, 👋, 📝, 👨‍💻 or 🆓, as I am not sure how the dolphin relates to contributing.

Reply:
well… dolphins are… dunno… I have no idea what I took when choosing these emojis :stuck_out_tongue_winking_eye: